composer.json

```
@@ -33,6 +33,10 @@
         "sebastian/environment": "^6.0.1",
         "spatie/laravel-ignition": "^2.3.0"
     },
+    "conflict": {
+        "laravel/framework": "<10.26.2 || >=11.0.0",
+        "phpunit/phpunit": "<10.3.5 || >=11.0.0"
```

We could also consider blocking `10.5.0` until we can ensure compatibility with the version:

```suggestion
        "phpunit/phpunit": "<10.3.5 || >=10.5.0"
```